Block/NodeType/Category.php

```diff
@@ -53,6 +53,10 @@ public function getImageThumb(int $nodeId)
      */
     protected function getCategoryId(int $nodeId)
     {
+        // fix to retrieve row_id for attribute value lookup
+        if ($category = $this->getCategory($nodeId)) {
```

Comment:
Will this load a category entity? Probably fine as long as it's not within a loop or anything

Reply:
the parent block class stores an array of categories in a 'categories' property - it's pulled from there if it exists